modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/LeftSidebar/LeftSidebar.tsx

```diff
@@ -55,10 +55,14 @@ export default function LeftSidebar({setShowModal}: LeftSidebarProps) {
 			};
 		});
 
-		setExpandedKeys(new Set(keys));
+		const selectedObjectFolderKey = keys.filter(
```

Comment:
@HenriquePereiraa If you're retrieving just one item, it's better to use a find than a filter then. Can you change it, please? Thanks!

Reply:
Ah, I get it!